lib/embedded-cli/embedded_cli.h

```diff
@@ -743,7 +743,7 @@ EmbeddedCli *embeddedCliNew(EmbeddedCliConfig *config) {
 
     bool allocated = false;
     if (config->cliBuffer == NULL) {
-        config->cliBuffer = (CLI_UINT *) malloc(totalSize); // malloc guarantees alignment.
+//        config->cliBuffer = (CLI_UINT *) malloc(totalSize); // malloc guarantees alignment.
```

Comment:
remove malloc/free usage since we use static allocation anyway. This prevent linker to try to find libc, heap start/end symbols.